game/src/Game/GameSession/BillboardManager.gd

```diff
@@ -0,0 +1,191 @@
+extends MultiMeshInstance3D
+
+#given a name: get the index for the texture in the shader
+# this is to reduce the number of magic indeces in the code
+# to get the proper billboard image
+var billboard_names = {}
```

Comment:
Can you make sure every GDScript `var`, `const` and function argument + return type has an explicit type given